lib/account/utils/profiles.dart

```diff
@@ -6,7 +6,7 @@ import 'package:thunder/core/auth/bloc/auth_bloc.dart';
 import 'package:thunder/thunder/bloc/thunder_bloc.dart';
 import 'package:thunder/account/widgets/profile_modal_body.dart';
 
-void showProfileModalSheet(BuildContext context) {
+void showProfileModalSheet(BuildContext context, {bool logOutOfActiveAccount = false}) {
```

Comment:
Thoughts on renaming `logOutOfActiveAccount` to `showLogoutDialog` or something similar? This seems like a more appropriate name since the user can still cancel the log out process

Reply:
Good idea, done!

Note I left the `_logOutOfActiveAccount()` method as is since it's private and to avoid confusion with the actual `showLogOutDialog` method.